DOCUMENTATION.md

```diff
@@ -42,6 +42,13 @@ Create an application. This function only works if you log in using credentials
 
 #### Raises:
     InvalidDeviceType: if device type is not supported.
+
+#### Examples:
+```python
+>>> resin.models.application.create('Edison','Intel Edison')
+Return: dict contains details of new application in the following format.
```

Comment:
This is redundant since you're documenting what the function returns. If you feel this adds something, prefer to improve on the "Returns" part.